src/views/List.jsx

```diff
@@ -134,26 +150,36 @@ export function List({ data, listToken, loading, confirmLogOut }) {
 				</>
 			)}
 			<ul>
-				{/* filter through groups array for each group to display by time frame */}
-				{groups.map((group) => {
-					return (
-						<section className={group.timeFrame}>
-							<h1>{group.timeFrame}</h1>
-							{searchResults
-								// within each group's filteredData, map through to each item to pass in as a prop
-								.filter((item) => group.filteredData(item))
-								.map((filteredItem) => {
-									return (
-										<ListItem
-											key={filteredItem.id}
-											item={filteredItem}
-											listToken={listToken}
-										/>
-									);
-								})}
-						</section>
-					);
-				})}
+				<div className="group-container">
+					{/* filter through groups array for each group to display by time frame */}
+					{groups.map((group) => {
+						return (
+							<section className={group.timeFrame}>
+								<div className="group-title">
+									<span>{group.image}</span>
+									<h1>{group.timeFrame}</h1>
+								</div>
+								<div className="list-titles">
+									<h3 className="item-title-1">Item</h3>
+									<h3 className="item-title-2">Bought</h3>
+									<h3 className="item-title-3">Delete</h3>
+								</div>
```

Comment:
Nice work handling the layout with this section, it looks amazing and great work on updating the groups config to include additional display information.

I saw some of the solutions you did to get this to all line up and I would recommend an HTML table for this layout and I think that would've helped with some of the alignment but honestly great work on figuring out how to get the alignment you wanted through CSS which is a huge skill